src/app/home/home.component.html

```diff
@@ -0,0 +1,2 @@
+<p class="welcome">Welcome to the MSAL.js Angular Quickstart!</p>
+<p>This sample demonstrates how to configure MSAL Angular, login, logout, protect a route, and acquire an access token for a protected resource such as the Microsoft Graph.</p>
```

Comment:
<p>This sample demonstrates how to configure MSAL Angular to login, logout, protect a route, and acquire an access token for a protected resource such as the Microsoft Graph.</p>